modules/json/pom.xml

```diff
@@ -18,6 +18,17 @@
     <dependency>
       <groupId>com.fasterxml.jackson.jaxrs</groupId>
       <artifactId>jackson-jaxrs-json-provider</artifactId>
+      <exclusions>
+        <exclusion>
+          <groupId>com.fasterxml.jackson.core</groupId>
+          <artifactId>jackson-databind</artifactId>
+        </exclusion>
+    </exclusions>
+    </dependency>
+    <dependency>
+      <groupId>com.fasterxml.jackson.core</groupId>
+      <artifactId>jackson-databind</artifactId>
+      <version>2.9.9.1</version>
```

Comment:
Please consider putting versions into `dependencyManagement` of BOM.
If you need to exclude it and then include it, it seems as the `jackson-jaxrs-json-provider` is shipping with and old vulnerable version of it. Is there no new version of `jackson-jaxrs-json-provider` with the new dependency version to make it consistent? IMHO we should simply update/override `jackson.version` in our BOM.

Reply:
I have found the latest version of bom for Jackson bom (2.10.0.pr1), they have just publish it. This jackson version excludes the vulnerable version of jar. I tried to update the Jackson.version in bom but it was not able to override that way. So have to put it in devon-modules , before spring-boot dependencies to override the jackson inside it

Reply:
Thanks for testing. Have you tried playing with the order of the BOM imports in our devon4j-bom? Maybe we can convince maven to use the desired version this way...
Do you know what `pr1` stands for? pre-release? Does not sound like a reliable version...

Reply:
I have removed the the pr1 version, and updated with 2.9.9.20190727.

Also I have tried changing the orders in devon4j-bom but with no success